include/odp/api/spec/packet_types.h

```diff
@@ -285,6 +285,17 @@ typedef enum odp_packet_reass_status_t {
 	ODP_PACKET_REASS_COMPLETE,
 } odp_packet_reass_status_t;
 
+/**
+ * Information about a completed reassembly
+ */
+typedef struct odp_packet_reass_info_t {
+	/** Number of fragments reassembled */
+	uint16_t num_frags;
+
+	/** IP packet length of the longest fragment that was reassembled */
+	uint16_t max_ip_len;
```

Comment:
Agreed to leave this field out for now, and add it later if applications need it.

Reply:
Removed in v2.